zwave_js_server/model/node.py

```diff
@@ -310,8 +310,38 @@ def receive_event(self, event: Event) -> None:
 
         self.emit(event.type, event.data)
 
+    async def _async_send_command(
+        self,
+        cmd: str,
+        require_schema: Optional[int] = None,
+        wait_for_result: bool = None,
+        **cmd_kwargs: Any,
+    ) -> Optional[Any]:
```

Comment:
Isn't the result always a dict?
```suggestion
    ) -> Optional[dict]:
```